src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -70,17 +71,17 @@ internal abstract class InstallingWorkloadCommand : WorkloadCommandBase
             _checkIfManifestExist = !(_printDownloadLinkOnly);      // don't check for manifest existence when print download link is passed
             _sdkVersion = WorkloadOptionsExtensions.GetValidatedSdkVersion(parseResult.GetValueForOption(InstallingWorkloadCommandParser.VersionOption), version, _dotnetPath, _userProfileDir, _checkIfManifestExist);
             _sdkFeatureBand = new SdkFeatureBand(_sdkVersion);
```

Comment:
The following code around line 134 should be changed to use `_sdkFeatureBand` instead of `_sdkVersion`:

```c#
var overlayProvider = new TempDirectoryWorkloadManifestProvider(extractedManifestsPath, _sdkVersion.ToString());
```

The constructor parameter name should also be changed.

Alternatively, the `TempDirectoryWorkloadManifestProvider` constructor could be changed to convert the version passed in into a feature band.

I'm not sure if this actually impacts anything or not.